### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_Keyboard:There is a trap present for the editors.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9365]
[Check out Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#Keyboard;#MAS2.1.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.1.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Click File then New Notebook.
3. Click Attach To and select a SQL connection.
4. Click + Text to add a text cell. Add some text.
5. Navigate using tab inside the code cell.

**Actual**
When enter is hit on +text button the focus moves into the editor with tab the navigation is stuck in the editor itself, thus the buttons present in the cells are not navigatable.
Same issue with the code button.

**Expected:**
There should be a key provided to move out of edit mode. The key should be specified visually on screen and on tab the button more options should be accessible.

**User Impact:**
Keyboard users will not be able to navigate out of the cell.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EcXq1cmRSHNArhEhvJ7vGAMBCA5_Fb7jMm2VfBdJbi2mpQ?e=4glp2J

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/azuredatastudio/issues/9405

**Comment by Roopesh-T** [comment]
Closing the bug, as the issue is not repro in 
![image](https://user-images.githubusercontent.com/86601127/218997079-9df3f3f8-8e7d-4d72-9f93-6c2a3a993229.png)
after activating the toggle tab focus. once the focus is entered into the text field and the editing is complete, we can press esc key to move out of the text box and can navigate outside with tab keys.
![9365_Closed](https://user-images.githubusercontent.com/86601127/218997466-fe70e426-22e7-46b9-960e-66662f9bb214.gif)

**GitHub Tag**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
